docs/community/README.md

```diff
@@ -8,39 +8,37 @@ import Grid, { Button } from '@site/src/components/Grid'
 
 # What is Radworks?
 
-Radworks is a community dedicated to cultivating internet freedom. In a world that almost entirely relies on software, maintaining the resilience and health of the free and open source ecosystem is more important than ever. We strive to demonstrate the viability — in cost, labor, quality, and resiliency — of open-source software and its development.
+Radworks is a community dedicated to **cultivating internet freedom**. In a world that almost entirely relies on software, maintaining the resilience and health of the free and open source ecosystem is more important than ever. We strive to demonstrate the viability — in cost, labor, quality, and resiliency — of open-source software and its development.
 
-Radworks is run by a decentralized governance system, powered by its community of users & builders, and represented by the $RAD token. Join us on our quest to create a better and free-er internet together! 
+Join us on our quest to create a better and free-er internet together! 
+
+## Our Purpose
+
+> Radworks funds new, resilient, permissionless technologies to cultivate internet freedom. We focus on funding and supporting the creation of censorship-resistant and decentralized technologies that empower builders and creators to collaborate. We strive to endorse and provide resources to builders of these technologies. 
+
+> Radworks is committed to transparency, inclusivity, and decentralized decision-making, ensuring that our community has a say in the direction of our efforts and the projects we support. Together, we can build a more decentralized and equitable future for software development. 
+
+> Our purpose lives [here](https://app.radicle.xyz/seeds/seed.radworks.org/rad:zPUsinVa3gP71g6Dt47LP76phAWd/tree/main/purpose.md).
+
+
+Radworks is run by a decentralized governance system, powered by its community of users & builders, and represented by the $RAD token. Read more about [Radworks governance](/community/governance-overview). 
 
 ## Our Projects
 Radworks currently supports the development of the following technologies:
 
 <Grid>
   <Button
-    style={{gridColumn: '1 / span 12'}}
     href="https://radicle.xyz"
     title="Radicle"
     cta="radicle.xyz"
   >
     Radicle is a sovereign peer-to-peer network for code collaboration, built on top of Git.
```

Comment:
These are the one-liners used on the website:

Radicle: A p2p network for code collaboration.
Drips: A toolkit for automatic, recurring payments

Do we want to use these or keep the ones here? I actually like the radicle description we currently have, but we should update the drips one.